src/common/arrow/src/arrow/bitmap/utils/mod.rs

```diff
@@ -68,6 +68,7 @@ pub fn set(byte: u8, i: usize, value: bool) -> u8 {
 /// panics if `i >= data.len() / 8`
 #[inline]
 pub fn set_bit(data: &mut [u8], i: usize, value: bool) {
+    assert!(i < data.len() / 8);
```

Comment:
I think we are not concerned about panicking in this situation, The bounds used are safe.